packages/terra-consumer-nav/src/Nav.jsx

```diff
@@ -4,7 +4,7 @@ import classNames from 'classnames/bind';
 import Button from 'terra-button';
 import IconClose from 'terra-icon/lib/icon/IconClose';
 import TerraPopup from 'terra-popup';
-import PopupHeights from 'terra-popup/lib/_PopupHeights';
+import PopupHeights from 'terra-popup/lib/PopupHeights';
```

Comment:
The new version of popup exposes this file in it's options, thus the name change from the private denoted '_'.
```
Popup.propTypes = propTypes;
Popup.defaultProps = defaultProps;
Popup.Opts = {
  attachmentPositions: ATTACHMENT_POSITIONS,
  heights: PopupHeights,
  widths: PopupWidths,
};

export default Popup;
```

Reply:
ahhh, that makes sense. Thanks for pointing this out!
I'll consume the variables that way instead of our hacky invasive non passive way.
we should be removing this anyways once we get the auto height + content that expands issue fix.